tsl/test/expected/chunk_api-11.out

```diff
@@ -80,17 +80,23 @@ CONTEXT:  JSON data, line 1:  {"time: [1515024000000000] "device...
 -- Valid chunk, but no permissions
 SET ROLE :ROLE_DEFAULT_PERM_USER_2;
 SELECT * FROM _timescaledb_internal.create_chunk('chunkapi',' {"time": [1515024000000000, 1519024000000000], "device": [-9223372036854775808, 1073741823]}', 'ChunkSchema', 'My_chunk_Table_name');
-ERROR:  must be owner of hypertable "chunkapi"
-SET ROLE :ROLE_DEFAULT_PERM_USER;
+ERROR:  permission denied for table "chunkapi"
+DETAIL:  Insert privileges required on "chunkapi" to create chunks.
 \set ON_ERROR_STOP 1
-\set VERBOSITY terse
--- Create a chunk that does not collide and with custom schema and name
+-- Try that granting insert on table allow create_chunk to be
+-- called. This will also create a chunk that does not collied and has
```

Comment:
```suggestion
-- Try that granting insert on table allows create_chunk to be
-- called. This will also create a chunk that does not collide and has
```